protocols/util/factories.py

```diff
@@ -0,0 +1,47 @@
+import factory.fuzzy
+import itertools
+from protocols.ga4gh_3_0_0 import Variant, Call
+from .avro_factory import FactoryAvro
+
+
+class GA4GHVariantFactory(FactoryAvro):
+    def __init__(self, *args, **kwargs):
+        super(GA4GHVariantFactory, self).__init__(*args, **kwargs)
+
+    class Meta:
+        model = Variant
+    start = factory.fuzzy.FuzzyInteger(1, 10000000)
```

Comment:
Can we use kwargs to pass ad hoc parameters???
I was thinking GA4GHVariantFactory(start=123456789, referenceBases='T', alternateBases='G', referenceName='22')